### PR TITLE
Write draft for behavior chapter

### PR DESCRIPTION
**Pull request by chhoumann**

### CODE REVIEW

File: Report/sections/1_analysis/Behavior.tex [hunk 1]
```diff
@@ -0,0 +1,19 @@
+\chapter{Behavior}\label{chap:behavior}
+
+As concluded in the last chapter, learning how to program for the first time is hard.
```
Comment (Pattrigue) [inline Report/sections/1_analysis/Behavior.tex:3]: difficult instead of hard?

File: Report/sections/1_analysis/Behavior.tex [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Therefore, a natural next question would be to ask how one could make it easier.
```
Comment (Pattrigue) [inline Report/sections/1_analysis/Behavior.tex:4]: The previous statement was not a question so saying "a natural **next** question" seems weird.

File: Report/sections/1_analysis/Behavior.tex [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+That is the topic this chapter will examine. \\
```
Comment (Pattrigue) [inline Report/sections/1_analysis/Behavior.tex:5]: maybe discuss or analyze instead of examine?

File: Report/sections/1_analysis/Behavior.tex [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Earlier, in \chapref{chap:analysis}, we presented \textit{deliberate practice} as an optimal way to learn.
+However, merely knowing how to learn is not the same as learning. \\
+
+For any behavior to occur, three elements must be present. Namely, motivation, ability to perform a given behavior, and a prompt \cite{bjfoggBehaviorModelPersuasive2009}. If any of those are not present, the behavior will not occur. \\
```
Comment (Pattrigue) [inline Report/sections/1_analysis/Behavior.tex:10]: This feels a little sudden and out of context. Maybe make a short introduction to behavioral psychology before presenting this as a cold hard fact?

File: Report/sections/1_analysis/Behavior.tex [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+Often, increasing a user's ability is the best choice to optimizing behavior performance \cite{bjfoggBehaviorModelPersuasive2009}. Therefore, we should aim to make it simpler for users to program games.
```
Comment (Pattrigue) [inline Report/sections/1_analysis/Behavior.tex:12]: for optimizing

Comment (Pattrigue) [reply Report/sections/1_analysis/Behavior.tex:12]: also, program games specifically or program in general? Here you say games specifically, do we want to home in on games specifically this early on?

File: Report/sections/1_analysis/Behavior.tex [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+Specifically, we will focus on the brain cycle aspect of learning how to program. If the user has to think hard to perform the behavior, it is not simple \cite{bjfoggBehaviorModelPersuasive2009}.
```
Comment (Pattrigue) [inline Report/sections/1_analysis/Behavior.tex:14]: The final sentence here feels out of place and would probably fit in better if it appeared a bit earlier.

File: Report/sections/1_analysis/Behavior.tex [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+So the question becomes: how do we make it easier for users to program games?
```
Comment (Pattrigue) [inline Report/sections/1_analysis/Behavior.tex:16]: Feels like this is repeating the first line without adding anything new, making it redundant 
"Therefore, a natural next question would be to ask how one could make it easier."

**Comment by chhoumann** [comment]
Irrelevant

**Pull request closed without merging**